### DISCUSSION TITLE
New Exercise: HAR(Human Activity Recognition) using LiDAR

### DISCUSSION THREAD

**Discussion by aastha51551** [JdeRobot/RoboticsAcademy#2902] [Ideas]
Hi Everyone!!
I had this idea in my mind for quite some time. 
The video-HAR technology approach suffers from occlusion problems, LiDAR technology can tackle these problems more efficiently as it uses 3D point cloud data rather than the 2D RGB images/videos utilized by video cameras.
Although we need to have different kind of models based on the no. of people, we can start at an individual level (one person in a room)

**Comment by jmplaza** [comment]
Sounds good to me. I suggest you (a) to work in this robotics application (=world in Gazebo Harmonic including lidar sensor and a moving person) out of RoboticsAcademy (RA). And then (b) develop a reference solution (take a look at [OpenPose](https://github.com/CMU-Perceptual-Computing-Lab/openpose), [this webpage](https://viso.ai/deep-learning/openpose/) and [this paper](https://www.mdpi.com/1424-8220/24/11/3427) for similar approaches). When you have everything working, then (c) integrate that exercise in RA.

Cheers,

**Reply by aastha51551** [reply]
Thanks for the directions. I’ll proceed with the tasks as outlined.
